examples/pipelines/controlnet-interior-design/components/download_images/fondant_component.yaml

```diff
@@ -0,0 +1,42 @@
+name: Download images
+description: Component that downloads images based on URLs
+image: download_images:latest
+
+input_subsets:
+  images:
+    fields:
+      url:
+        type: utf8
+
+output_subsets:
+  images:
+    fields:
+      data:
+        type: binary
+      width:
+        type: int16
+      height:
+        type: int16
+
+args:
+  timeout:
+    description: Maximum time (in seconds) to wait when trying to download an image
+    type: int
+  retries:
+    description: Number of times to retry downloading an image if it fails.
+    type: int
+  image_size:
+    description: Size of the images.
```

Comment:
maybe make the description more explicit or the argument. It's the size to resize the image to